jni/meson.build

```diff
@@ -15,6 +15,8 @@ endif
 if tflite_dep.found()
   and_conf.set('MESON_HAS_TFLITE', 1)
   and_conf.set('MESON_TFLITE_ROOT', tflite_root)
+else
+  and_conf.set('MESON_HAS_TFLITE', 0)
```

Comment:
It could be good if we provide a message saying which feature will not be supported if this option is disabled.

Reply:
updated as per suggestion! https://github.com/nnstreamer/nntrainer/pull/1685/files#r740688882